sql.txt

```diff
@@ -1,14 +1,28 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (SupplierID, CategoryID) VALUES (1, 3);
+INSERT INTO Products (SupplierID, CategoryID) VALUES (2, 1);
+INSERT INTO Products (SupplierID, CategoryID) VALUES (2, 4);
+
```

Comment:
This works. You could also have done something like below:
INSERT INTO Products 
VALUES 
(NULL, "Hot Chesse", 1, 1, "1 kg pkg", 30.85),
(NULL, "Nachos", 1, 1, "2kg pkg", 60.55),
(NULL, "Taco Tortilla", 1, 1, "5 kg pkg", 200);